var/spack/repos/builtin/packages/openscenegraph/package.py

```diff
@@ -12,41 +11,59 @@ class Openscenegraph(CMakePackage):
        that's used in a variety of visual simulation applications."""
 
     homepage = "http://www.openscenegraph.org"
-    url      = "https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.6.4.tar.gz"
+    git_url  = "https://github.com/openscenegraph/OpenSceneGraph.git"
```

Comment:
```suggestion
    git  = "https://github.com/openscenegraph/OpenSceneGraph.git"
```
If you do this, you don't need to specify `git=git_url` for every version. Although I'm not sure how I feel about switching from `url` to `git`. Checksums are more reliable than tags which can change.

Reply:
I wasn't aware git packages could have the URL specified that way; that makes things much simpler! As for `url` versus `git` specification, I rather like using `git` tags for the readability and ease of use benefits, but you do have a good point about the reliability benefits of checksums. I consider reliability to be of much higher importance for something like a Spack package, so I'll revert this change and add checksums for the new versions.